.travis.yml

```diff
@@ -2,24 +2,37 @@ language: python
 sudo: false
 matrix:
   include:
-    - python: 3.5
-      env: TOXENV=py35
-    - python: 3.5
-      env: TOXENV=py35-pure
-env:
-    - TOXENV=py27
-    - TOXENV=py27-minimal
-    - TOXENV=py27-pure
-    - TOXENV=pypy
-    - TOXENV=py33
-    - TOXENV=py34
-    - TOXENV=py34-pure
-    - TOXENV=pypy3
-    - TOXENV=coverage
-    - TOXENV=docs
-install:
-    - pip install tox
+    - python: 2.7
+      env: DOCS=1
+      # The doctests only run on Python 2.7
+    - python: 2.7
+      env: MINIMAL="-t !persistentregistry -t !security"
+python:
+  - 2.7
+  - 3.4
+  - 3.5
+  - 3.6
+  - pypy-5.6.0
+  - pypy3.3-5.5-alpha
+
 script:
-    - tox
+  - coverage run -m zope.testrunner --test-path=src $MINIMAL
+  - if [[ -n "$DOCS" ]]; then sphinx-build -b html    -d docs/_build/doctrees docs docs/_build/html; fi
+  - if [[ -n "$DOCS" ]]; then coverage run `which sphinx-build` -b doctest -d docs/_build/doctrees docs docs/_build/doctest; fi
+
+after_success:
+  - coveralls
 notifications:
-    email: false
+  email: false
+
+install:
+  - pip install -U pip setuptools
+  - pip install -U coveralls coverage
+  - if [[ -n "$MINIMAL" ]]; then pip install -U -e ".[mintests]"; fi
+  - if [[ -z "$MINIMAL" ]]; then pip install -U -e ".[test,docs]"; fi
```

Comment:
I'd like to do a subsequent PR that gets the doctests running in Python 3 and turn them on by default.